<h1 align="center">Hi ðŸ‘‹, I'm Tran Ngoc Tam</h1>

---

# Latest Blog Posts 
<!-- BLOG-POST-LIST:START -->
- [Don&#39;t Learn to Code Until You Read This](https://dev.to/thetechpreneur/dont-learn-to-code-until-you-read-this-1ohn)
- [Pattern Printing Series: Alphabet Patterns Explained with Logic](https://dev.to/datatoinfinity/pattern-printing-series-alphabet-patterns-explained-with-logic-5enl)
- [Mark Rober: Making MrBeast Drink His Own Pee &lpar;w/Science&rpar;](https://dev.to/maker_youtube/mark-rober-making-mrbeast-drink-his-own-pee-wscience-cm0)
- [Class, Object, Instance](https://dev.to/emmakang123/class-object-instance-3pel)
- [IGN: EA Sports Madden NFL 26 - Official Live-Action Launch Trailer](https://dev.to/gg_news/ign-ea-sports-madden-nfl-26-official-live-action-launch-trailer-3npk)
<!-- BLOG-POST-LIST:END -->

---

# Latest stack oveflow activity
<table>
  <tr><th>Title</th><th>Link</th></tr>
  <!-- STACKOVERFLOW:START --><tr><td>getting &quot;Fatal error: Cannot declare class phpbb\di\pass\collection_pass, because the name is already in use&quot; by instaling phpbb3 on website</td><td>https://stackoverflow.com/questions/79722722/getting-fatal-error-cannot-declare-class-phpbb-di-pass-collection-pass-becaus</td></tr><tr><td>How to handle a external service timeout with php fpm in web app</td><td>https://stackoverflow.com/questions/79722576/how-to-handle-a-external-service-timeout-with-php-fpm-in-web-app</td></tr><tr><td>Forge CLI: Logged in successfully, but all commands return Your API Token is invalid</td><td>https://stackoverflow.com/questions/79722535/forge-cli-logged-in-successfully-but-all-commands-return-your-api-token-is-inv</td></tr><tr><td>Uncaught &lpar;in promise&rpar; TypeError: NetworkError when attempting to fetch resource in Laravel 12 + Livewire V3 + Vite setup</td><td>https://stackoverflow.com/questions/79722462/uncaught-in-promise-typeerror-networkerror-when-attempting-to-fetch-resource</td></tr><tr><td>Script in Python CVE-2025-49113 Not receiving the connection</td><td>https://stackoverflow.com/questions/79722326/script-in-python-cve-2025-49113-not-receiving-the-connection</td></tr><!-- STACKOVERFLOW:END -->
</table>

---


